pkg/catalog/catalog.go

```diff
@@ -102,9 +104,10 @@ func getRegistryDevfiles(registry Registry) ([]DevfileComponentType, error) {
 	}
 	registry.URL = URL
 	indexLink := registry.URL + indexPath
-	jsonBytes, err := util.HTTPGetRequest(indexLink)
+	token, _ := keyring.Get(credentialPrefix+registry.Name, "default")
+	jsonBytes, err := util.HTTPGetRequest(indexLink, token)
 	if err != nil {
-		return nil, errors.Wrapf(err, "Unable to download the devfile index.json from %s", indexLink)
+		return nil, err
```

Comment:
Yeah, the reason why I remove is because I find the error message returns from `utilHTTPGetRequest` is specific enough, so removing that can make the error message is not that verbose, but I can add it back to make it more specific.